specs/core/0_beacon-chain.md

```diff
@@ -1766,7 +1767,8 @@ def process_deposit(state: BeaconState, deposit: Deposit) -> None:
     validator_pubkeys = [v.pubkey for v in state.validator_registry]
     if pubkey not in validator_pubkeys:
         # Verify the deposit signature (proof of possession)
-        if not bls_verify(pubkey, signing_root(deposit.data), deposit.data.signature, get_domain(state, DOMAIN_DEPOSIT)):
+        # Note: deposits are valid regardless of fork version, hence the type is passed directly as domain.
+        if not bls_verify(pubkey, signing_root(deposit.data), deposit.data.signature, DOMAIN_DEPOSIT):
```

Comment:
Type mismatch

Reply:
It's a verbosity thing. In python it's just an integer. And it aligns to the lower part. And there's no way to explicitly cast from uint32 to uint64. Open for suggestions.